minecode/visitors/openssl.py

```diff
@@ -88,3 +90,26 @@ def get_uris(self, content):
                 yield URI(uri=url, source_uri=self.uri, package_url=package_url, date=date, file_name=file_name, size=size)
             else:
                 yield URI(uri=url, source_uri=self.uri, date=date, size=size)
+
+# Indexing OpenSSL PURLs requires a GitHub API token.
+# Please add your GitHub API key to the `.env` file, for example: `GH_TOKEN=your-github-api`.
+@priority_router.route('pkg:openssl/openssl@.*')
```

Comment:
Is there a test for openssl purls or is it handled by `test_map_fetchcode_supported_package`?

Reply:
We have test for OpenSSL PURLs here https://github.com/nexB/fetchcode/blob/ff0bad32881464afbfa6f70a65909ed0170e94e5/tests/test_package.py#L503-L522